seed/index.js

```diff
@@ -1,12 +1,24 @@
 require("dotenv").config()
 
 const SpoofItem = require("./../api/models/spoofitem")
-const mongoose = require("mongoose")
-mongoose.connect(process.env.MONGODB_URI, {
-  useMongoClient: true
+const {mongoose} = require('./../config/mongoose')
+const Counter = require("./../api/models/counter")
+
+
+//Clear Collections to start new + add counter
+SpoofItem.remove({}).then((result) => {
+  console.log('spoof items cleared')
+})
+
```

Comment:
I don't think `Counter.remove` is synchronous. The create query can execute before the DB has been cleared.